pkgs/development/libraries/ncurses/default.nix

```diff
@@ -12,12 +12,15 @@
 , testers
 }:
 
-stdenv.mkDerivation (finalAttrs: {
-  version = "6.4";
+stdenv.mkDerivation (finalAttrs: let
+ semver = "6.4";
+ release = "20221231";
+in {
+  version = "${semver}.${release}";
   pname = "ncurses" + lib.optionalString (abiVersion == "5") "-abi5-compat";
 
   src = fetchurl {
-    url = "https://invisible-island.net/archives/ncurses/ncurses-${finalAttrs.version}.tar.gz";
+    url = "https://invisible-island.net/archives/ncurses/ncurses-${semver}.tar.gz";
```

Comment:
I believe you can do `lib.versions.majorMinor version` instead of having a `semver`.